ui/raidboss/data/00-misc/test.ts

```diff
@@ -362,10 +362,31 @@ const triggerSet: TriggerSet<Data> = {
         },
       },
     },
+    {
+      id: 'Test OutputStrings',
+      type: 'GameLog',
+      netRegex: {
+        line: 'cactbot test outputStrings',
+        code: Util.gameLogCodes.echo,
+        capture: false,
+      },
+      infoText: (data, _matches, output) => {
+        // TODO: This doesn't work unless you're in a party, because party tracker is empty.
```

Comment:
Maybe `partyMemberParam` could handle this case? Or is that too much plumbing to get the current player in there too? (oopsy already does this dance in `PlayerStateTracker.OnPartyChanged`)

Reply:
I feel like if we're going to fix this cactbot-side, we should do so at a higher level e.g. in `popup-text.ts`'s party change handler?

I don't see the point in fixing the problem here when it's also a problem elsewhere, e.g. in any trigger that's role-based which you might encounter while doing content solo unsync'd.

But overall I'd rather fix this upstream in OverlayPlugin or in the base FFXIV_ACT_Plugin.

Reply:
Makes sense. Let's defer this for now then. Fixing this in OP or FFXIV_ACT_Plugin would definitely be much better.